docs/components/details.md

```diff
@@ -30,6 +30,34 @@ const App = () => (
 
 ## Examples
 
+### Details with shadow
+
+Use the `shadow` property to add a box shadow to the details card. The shadow matches the `Card` component's shadow.
+
+```html preview
+<sl-details summary="Details with shadow" shadow>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+  aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+</sl-details>
+```
+
+```pug slim
+sl-details summary="Details with shadow" shadow="true"
```

Comment:
For attributes that we want to show up as a single word (i.e. `shadow` rather than `shadow="true"`), we should use `true` (boolean) instead of `"true"` (string). Slim will then render the attribute without a string value, and won't render the attribute at all if the boolean is false. 

This is important because, while setting `shadow="true"` will appear to work, setting `shadow="false"` (with a string) will still register as `true` as far as the component is concerned, since the component is just looking for the _presence_ of the attribute, rather than the _value_.

```suggestion
sl-details summary="Details with shadow" shadow=true
```